src/logic/PlayerController.cpp

```diff
@@ -835,8 +835,7 @@ void PlayerController::onUpdateByInput(float deltaTime)
             }
             else
             {
-                // This happens when NPC enters area that should not be accessible
```

Comment:
I'd like to keep the log message as long as it's not clear why materials are not found, we should probably add a 'trace' log level (which is disabled by default) for such situations ...